ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -58,12 +60,12 @@ function (
         for (i = 0; i < blocks.length; i++) {
             block = blocks[i];
 
-            if (i === 1 || i === 6) {
+            if (i === 2 || i === 7) {
                 numberOfMpus++;
                 mpu = createMpu(numberOfMpus);
 
                 if (block.nextSibling) {
-                    block.parentNode.insertBefore(mpu, block.nextSibling);
+                    block.parentNode.insertBefore(mpu, block);
```

Comment:
Ok great I see how it works with the new querySelectorAll If I'm correct this is how it'd add an Advert (A) if there's a epic (C) behind a block (B): 

BBBBBBBB ----> BB**A**BBBBBB
BCBBBBBBB ----> BCBB**A**BBBBB**A**B
BBCBBBBBB ----> BBCB**A**BBBBB**A**B

This looks good to me!